src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Input/CursorConverter.cs

```diff
@@ -53,35 +52,29 @@ public override bool CanConvertTo(ITypeDescriptorContext context, Type destinati
             return false;
         }
 
-        /// <summary>
-        ///     Gets the public/static properties of the Cursors class
-        /// </summary>
-        /// <returns>PropertyInfo array of the objects properties</returns>
-        private PropertyInfo[] GetProperties()
-        {
-            return typeof(Cursors).GetProperties(BindingFlags.Public | BindingFlags.Static);
-        }
-
         /// <summary>
         ///     StandardValuesCollection method override
         /// </summary>
         /// <param name="context">ITypeDescriptorContext</param>
         /// <returns>TypeConverter.StandardValuesCollection</returns>
         public override TypeConverter.StandardValuesCollection GetStandardValues(ITypeDescriptorContext context)
         {
-            if(this._standardValues == null)
+            if(_standardValues is null)
             {
-                ArrayList list1 = new ArrayList();
-                PropertyInfo[] infoArray1 = this.GetProperties();
-                for(int num1 = 0; num1 < infoArray1.Length; num1++)
+                PropertyInfo[] properties = typeof(Cursors).GetProperties(BindingFlags.Public | BindingFlags.Static);
```

Comment:
Personally, I prefer to keep the old method. Thank you.

Reply:
@lindexi I'm sorry but I do not see a reason for the method. I actually had to go and find out what the method does originally, it is only used in one place and introduces imo unnecessary code complexity for a one-liner.

Reply:
Does it not get inlined? Not everything is about code complexity, there is also legibility and maintainability. This one did raise an eyebrow but I don't feel that strong about it.

Reply:
@miloush It will get inlined, it is rather small and can even be made static iirc. But my point is that since this is not a function on Cursor class but in the converter, called as "this.GetProperties()", I had to go and check what this function does previously, everyone will going through the code. That is what I meant when I said code complexity. Not the actual codegen — that will be identical.

Even though you see it returns PropertyInfo array, you don't know **which flags** and **which properties** as in reflection context it can mean anything. Does it filter it afterwards? Questions. So that is my reason for putting this one inside.